setup.py

```diff
@@ -40,8 +40,8 @@
         "pytest-runner",
     ],
     use_scm_version={"write_to": "genome_grist/version.py"},
-    install_requires=["snakemake>=5.10", "click>=7", "lxml==4.6.1", "pandas"],
-    tests_require=["pytest>=5.1.2", "pytest-dependency"],
+    install_requires=["snakemake>=5.10", "click>=7,<8", "lxml==4.6.1",
```

Comment:
should `snakemake>=5.10` be changed to `snakemake=6.4.1` to match `environment.yml`?

Reply:
ooh, good catch, thank you :)

Reply:
fixed in ac89910